modules/button/css/lib/button.scss

```diff
@@ -22,27 +22,30 @@ $_wdc-btn-caret-down-icon-hover: svg-load(
 @mixin wdc-btn() {
   @include wdc-type-body();
   @include wdc-type-variant-btn();
-  @include wdc-btn-box-shadow($wdc-btn-border-color);
   display: inline-block;
   text-decoration: none;
+  font-family: $wdc-btn-font-family;
+  font-weight: $wdc-btn-font-weight;
   min-width: $wdc-btn-min-width-l;
   max-width: $wdc-btn-max-width-l;
   height: $wdc-btn-height-l;
-  line-height: $wdc-btn-height-l;
   padding: $wdc-btn-padding-l;
   background-color: $wdc-btn-bg;
-  border: none;
+  border: 1px solid $wdc-btn-border;
   color: $wdc-btn-color;
   border-radius: $wdc-btn-border-radius;
-  transition: 0.2s background, 0.2s border-color, 0.2s box-shadow;
+  transition: all 120ms linear;
```

Comment:
Because of the `focusRing` helper, this doesn't actually match. Can you do `all 120ms, box-shadow 100ms` instead? (linear easing is the default)